en/getting_started/tunes.md

```diff
@@ -226,8 +229,11 @@ Your browser does not support the audio element.
   <source src="../../assets/tunes/15_home_set_tune.mp3" type="audio/mpeg">
 Your browser does not support the audio element.
 </audio>
+<!-- 13, HOME_SET (previously id 15) -->
 
 - Home position initialised (first time only).
 
+#### Power Off Tune
```

Comment:
@julianoes Note, missing tune I "think" you added. I can do this if you can't, but not until next week.

Reply:
It currently only applies to Mantis.

Reply:
Sure, but one day it will apply to more. Added in #1791